packages/gestalt/src/IconButton.css

```diff
@@ -13,6 +13,16 @@
   background-color: var(--color-transparent);
 }
 
+.label {
+  font-family: var(--sema-font-family-UI-XS, "Pin Sans");
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  letter-spacing: var(--sema-font-letterspacing-UI-XS, 0);
+  line-height: var(--sema-font-lineheight-UI-XS, 16px);
```

Comment:
you don't need 2 arguments 
```
var(--sema-font-lineheight-UI-XS, 16px);
var(--sema-font-lineheight-UI-XS);
```

Reply:
rebase cos ravi changed many token names

Reply:
Fixed!